app/src/main/java/com/rockthevote/grommet/ui/registration/RegistrationDataTransformer.kt

```diff
@@ -0,0 +1,353 @@
+            ),
+            voterIds = buildVoterIds(),
+            contactMethods = buildContactMethods(),
+            additionalInfo = buildAdditionalInfo()
+        )
+
+    }
+
+    private fun buildRegistrationHelper(): RegistrationHelper? {
+        if (!assistanceData.hasSomeoneAssisted) return null // TODO is this correct, or should this be based on the canvasser and always non-null?
+
+        val helperPhone = assistanceData.helperPhone ?: throw InvalidRegistrationException(
+            "helperPhone is null",
+            R.string.assistants_phone_number_is_empty
+        )
+
+        return RegistrationHelper(
+            registrationHelperType = "assistant", // TODO is this a default value?
+            name = assistanceData.helperName?.toApiName(),
+            address = assistanceData.helperAddress?.toApiAddressData(),
+            contactMethods = listOf(
+                ContactMethod(
+                    type = "phone",
+                    value = helperPhone,
+                    capabilities = listOf("voice") // TODO should capabilties be more here? IIRC, this is only VOICE from [ApiContactMethod]
+                )
+            )
+        )
+    }
+
+    private fun buildVoterClassifications(): List<VoterClassification> {
+        val is18OnElectionDay = VoterClassification(
+            type = "eighteen_on_election_day",
+            assertion = newRegistrationData.is18OrOlderByNextElection
+        )
+
+        val isUsCitizen = VoterClassification(
+            type = "united_states_citizen",
+            assertion = newRegistrationData.isUsCitizen
+        )
+
+        val sendCopyInEmail = VoterClassification(
+            type = "send_copy_in_mail", // TODO Is this "I give permission to OSET Org to send me news and updates?" It's looking like it may not be, that might be "opt_in_email"? Where does this come from?
+            assertion = additionalInfoData.hasOptedIntoNewsUpdates
+        )
+
+        val agreedToDeclaration = VoterClassification(
+            type = "agreed_to_declaration", // TODO I can't find where this comes from, but it's in the API docs
+            assertion = true // TODO Defaulting to this until source is confirmed
+        )
+
+        // TODO, this isn't in the docs, but is in the old VoterClassification, should it be?
+        val politicalPartyChanged = VoterClassification(
+            type = "political_party_change",
+            assertion = additionalInfoData.hasChangedPoliticalParty
+        )
+
+        return listOfNotNull(
+            is18OnElectionDay,
+            isUsCitizen,
+            sendCopyInEmail,
+            agreedToDeclaration,
+            politicalPartyChanged
+        )
+    }
+
+    private fun buildVoterIds(): List<VoterId> {
+        val driversLicense = VoterId(
+            type = "drivers_license",
+            // TODO JSON example has this split with spaces, but app doesn't format it, does that matter? eg 99999999 is 99 999 999 in JSON example
```

Comment:
Shouldn't but I'll ask, good catch

Reply:
I see we're still waiting on clarification for this. To move on, I'll create a ticket and move it onto the board just so this isn't lost.

Reply:
https://mechanical-man.atlassian.net/browse/GROM-132